Source/MethodCall.cs

```diff
@@ -253,67 +245,44 @@ public void EvaluatedSuccessfully()
 
 		public virtual void Execute(ICallContext call)
 		{
-			this.Invoked = true;
```

Comment:
The `Invoked` property's value now gets computed from `callCount`, no need to set it anymore.